test/test.html

```diff
@@ -33,6 +33,7 @@
   </script>
 
   <script src="/node_modules/promise-polyfill/promise.js"></script>
+  <script src="/node_modules/abortcontroller-polyfill/dist/abortcontroller.js"></script>
```

Comment:
This can now import `abortcontroller-polyfill/dist/abortcontroller-polyfill-only` since v1.1.8